c8/fabric.py

```diff
@@ -668,10 +668,13 @@ def update_collection_properties(self, collection_name, has_stream=None, wait_fo
 
         :param collection_name: Collection name.
         :type collection_name: str | unicode
+<<<<<<< HEAD
+=======
```

Comment:
you will need to remove this probably came when updating branch

Reply:
+1